posix.mak

```diff
@@ -375,6 +377,16 @@ unittest/%.run : $(ROOT)/unittest/test_runner
 %.test : $(LIB)
 	$(MAKE) -f $(MAKEFILE) $(addsuffix .test,$(patsubst %.d,%,$(wildcard $*/*)))
 
+%.debug_with_debugger : %.d $(LIB)
+	$(DMD) $(DFLAGS) -main -unittest $(LIB) -defaultlib= -debuglib= $(LINKDL) $<
```

Comment:
Hmm, if you already link with the library, what is the effect of specifying an extra Phobos .d file? Won't it have the same code that's in the library anyway?